cadc-access-control-server/build.gradle

```diff
@@ -32,10 +32,11 @@ dependencies {
     compile 'org.opencadc:cadc-access-control:[1.1.23,2.0)'
     compile 'org.opencadc:cadc-util:[1.9.2,)'
     compile 'org.opencadc:cadc-log:[1.1.0,)'
-    compile 'org.opencadc:cadc-registry:[1.6,2.0)'
     compile 'org.opencadc:cadc-uws:[1.0,)'
     compile 'org.opencadc:cadc-uws-server:[1.2.4,)'
 
+    runtime 'org.opencadc:cadc-registry:[1.6,2.0)'
```

Comment:
seems odd to declare a runtime dependency in a library... should only be needed for a lib that provides a plugin not needed to compile

Reply:
Removed line 38.